nearlib/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "nearlib",
   "description": "Javascript library to interact with near blockchain",
-  "version": "0.2.4",
+  "version": "0.2.5",
```

Comment:
As I understand there are changes to API. This means version needs to be `0.3.0`, see http://semver.org/